zerver/views/webhooks/trello/card_actions.py

```diff
@@ -0,0 +1,197 @@
+    }
+    return TRELLO_SUBJECT_TEMPLATE.format(**data)
+
+def get_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    message_body = ACTIONS_TO_FILL_BODY_MAPPER.get(action_type)(payload, action_type)
+    creator = payload.get('action').get('memberCreator').get('fullName')
+    return TRELLO_MESSAGE_TEMPLATE.format(full_name=creator, rest=message_body)
+
+def get_added_checklist_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    data = {
+        'checklist_name': get_action_data(payload).get('checklist').get('name'),
+    }
+    return fill_appropriate_message_content(payload, action_type, data)
+
+def get_added_attachment_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    data = {
+        'attachment_url': get_action_data(payload).get('attachment').get('url'),
+        'attachment_name': get_action_data(payload).get('attachment').get('name'),
+    }
+    return fill_appropriate_message_content(payload, action_type, data)
+
+def get_updated_card_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    data = {
+        'card_name': get_card_name(payload),
+        'old_list': get_action_data(payload).get('listBefore').get('name'),
+        'new_list': get_action_data(payload).get('listAfter').get('name'),
+    }
+    return fill_appropriate_message_content(payload, action_type, data)
+
+def get_renamed_card_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    data = {
+        'old_name': get_action_data(payload).get('old').get('name'),
+        'new_name': get_action_data(payload).get('card').get('name'),
+    }
+    return fill_appropriate_message_content(payload, action_type, data)
+
+def get_added_label_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    data = {
+        'color': get_action_data(payload).get('value'),
+        'text': get_action_data(payload).get('text'),
+    }
+    return fill_appropriate_message_content(payload, action_type, data)
+
+def get_managed_member_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    data = {
+        'member_name': payload.get('action').get('member').get('fullName')
+    }
+    return fill_appropriate_message_content(payload, action_type, data)
+
+def get_managed_due_date_body(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    date = datetime.strptime(get_action_data(payload).get('card').get('due'), "%Y-%m-%dT%H:%M:%S.000Z")
+    data = {
+        'due_date': date.strftime('%m/%d/%Y %I:%M%p')
+    }
+    return fill_appropriate_message_content(payload, action_type, data)
+
+def get_body_by_action_type_without_data(payload, action_type):
+    # type: (Dict[str, Any], str) -> str
+    return fill_appropriate_message_content(payload, action_type)
+
+def fill_appropriate_message_content(payload, action_type, data=None):
+    # type: (Dict[str, Any], str, Any) -> str
```

Comment:
`action_type` should be `text_type` and `data` should be `Optional[MutableMapping[str, Any]]`.